### DISCUSSION TITLE
Poor quality after download

### DISCUSSION THREAD

**Discussion by Alf1973** [aclap-dev/video-downloadhelper#939] [Help and Q&A]
### Your question

I cant download with 1080 resolutoin. Have tried couple sources. Youtube gives 400 kbps and 480p, and one other even less. How can i download with decent quality? I have bought premium in firefox.

### Addon & CoApp version and details

version: 9.0.2.12
channel: stable
build date: 2024-05-30
build options: linuxlic: false, noyt: false, browser: firefox
lang: fi
license: accepted
key (only 16 first characters): a7033f57307f4e0a
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:127.0) Gecko/20100101 Firefox/127.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\kpkir",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
#943
